lib/vagrant-libvirt/action/create_networks.rb

```diff
@@ -100,8 +100,10 @@ def lookup_bridge_by_name(bridge_name)
         # match what was configured in the vagrantfile
         # since we always enable dhcp for the management network
         # this ensures we wont start a vm vagrant cant reach
+        # Allow the situation where DHCP is not requested (:libvirt__dhcp_enabled == false)
+        # but where it is enabled on the virtual network
         def verify_dhcp
-          unless @options[:dhcp_enabled] == @interface_network[:dhcp_enabled]
+          if @interface_network[:dhcp_enabled] == true && @options[:dhcp_enabled] == false
```

Comment:
hello @epienbroek  .
can you explain more ?
I not understand what and for what you change.